packages/web-components/src/horizontal-scroll/horizontal-scroll.styles.ts

```diff
@@ -0,0 +1,132 @@
+import { css } from '@microsoft/fast-element';
+import { DirectionalStyleSheetBehavior, display } from '@microsoft/fast-foundation';
+
+const ltrActionsStyles = css`
+  .scroll-prev {
+    right: auto;
+    left: 0;
+  }
+
+  .scroll.scroll-next::before,
+  .scroll-next .scroll-action {
+    left: auto;
+    right: 0;
+  }
+
+  .scroll.scroll-next::before {
+    background: linear-gradient(to right, transparent, var(--scroll-fade-next));
+  }
+
+  .scroll-next .scroll-action {
+    transform: translate(50%, -50%);
+  }
+`;
+
+const rtlActionsStyles = css`
+  .scroll.scroll-next {
+    right: auto;
+    left: 0;
+  }
+
+  .scroll.scroll-next::before {
+    background: linear-gradient(to right, var(--scroll-fade-next), transparent);
+    left: auto;
+    right: 0;
+  }
+
+  .scroll.scroll-prev::before {
+    background: linear-gradient(to right, transparent, var(--scroll-fade-previous));
+  }
+
+  .scroll-prev .scroll-action {
+    left: auto;
+    right: 0;
+    transform: translate(50%, -50%);
+  }
+`;
+
+/**
+ * Styles used for the flipper container and gradient fade
+ * @public
+ */
+export const ActionsStyles = css`
+  .scroll-area {
+    position: relative;
+  }
+
+  div.scroll-view {
+    overflow-x: hidden;
+  }
+
+  .scroll {
+    bottom: 0;
+    pointer-events: none;
+    position: absolute;
+    right: 0;
+    top: 0;
+    user-select: none;
+    width: 100px;
+  }
+
+  .scroll.disabled {
+    display: none;
+  }
+
+  .scroll::before,
+  .scroll-action {
+    left: 0;
+    position: absolute;
+  }
+
+  .scroll::before {
+    background: linear-gradient(to right, var(--scroll-fade-previous), transparent);
+    content: '';
+    display: block;
+    height: 100%;
+    width: 100%;
+  }
+
+  .scroll-action {
+    pointer-events: auto;
+    right: auto;
+    top: 50%;
+    transform: translate(-50%, -50%);
+  }
+`.withBehaviors(new DirectionalStyleSheetBehavior(ltrActionsStyles, rtlActionsStyles));
+
+/**
+ * Styles handling the scroll container and content
+ * @public
+ */
+export const HorizontalScrollStyles = css`
+  ${display('block')} :host {
+    --scroll-align: center;
+    --scroll-item-spacing: 5px;
```

Comment:
Can we make this 4px by default as that’s more “on ramp” for Fluent?

Reply:
Done